src/borg/archiver.py

```diff
@@ -1322,6 +1322,8 @@ def build_parser(self, prog=None):
                                   help='set umask to M (local and remote, default: %(default)04o)')
         common_group.add_argument('--remote-path', dest='remote_path', metavar='PATH',
                                   help='set remote path to executable (default: "borg")')
+        common_group.add_argument('--remote-ratelimit', dest='remote_ratelimit', type=int, metavar='rate',
+                                  help='set remote network rate limit in kbyte/s (default: unlimited)')
```

Comment:
yes, i should either document that or implement limiting for the other side too. I tend to just document it for now. Because i believe one of the main use cases is asymmetrical (slow) lines and it’s not too clear what would be what users want here. Also i expect the main use case where the user is not actively waiting for borg to be the create case.